.github/workflows/blossom-ci.yml

```diff
@@ -56,7 +56,6 @@ jobs:
       wbo4958,\
       wjxiz1992,\
       sperlingxx,\
-      pxLi,\
```

Comment:
Should target to branch24.02, and auto-merge will update the change to branch-24.04

Reply:
This can apply to 24.04 onwards.